src/users/serializers/update-user-options.serializer.ts

```diff
@@ -2,7 +2,13 @@ import { SerializedUpdateUserOptions, UpdateUserOptions } from '../interfaces';
 
 export const serializeUpdateUserOptions = (
   options: UpdateUserOptions,
-): SerializedUpdateUserOptions => ({
-  first_name: options.firstName,
-  last_name: options.lastName,
-});
+): SerializedUpdateUserOptions => {
+  if (Object.keys(options).length === 0) {
```

Comment:
nit: More idiomatic to write the length check like this:

```suggestion
  if (!Object.keys(options).length) {
```